src/main.ts

```diff
@@ -13,10 +13,29 @@ export default class GraphvizPlugin extends Plugin {
     await this.loadSettings();
     this.addSettingTab(new GraphvizSettingsTab(this));
     const processors = new Processors(this);
-    console.debug('add processor for dot');
-    this.registerMarkdownCodeBlockProcessor('dot', processors.imageProcessor.bind(processors));
+    const d3Sources = ['https://d3js.org/d3.v5.min.js',
+      'https://unpkg.com/@hpcc-js/wasm@0.3.11/dist/index.min.js',
+      'https://unpkg.com/d3-graphviz@3.0.5/build/d3-graphviz.js'];
+
+
+    this.app.workspace.onLayoutReady(() => {
+      switch (this.settings.renderer) {
+        case 'd3_graphviz':
+          for (const src of d3Sources) {
+            const script = document.createElement('script');
```

Comment:
There should be a better way to load those 3 js resources rather than using an `script` element. I did some research and did try to include it as a build dependency, but I'm simply too newbie on the subject. So consider it as a hack.